### PR TITLE
feat: added pages

### PR DESCRIPTION
**Pull request by Avin008**
**Pages Includes:**

- Home
- Products
- Cart
- Login
- Signup
- Wishlist
- index.js

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
This is looking good you can merge

File: src/pages/Cart/Cart.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import { Link } from "react-router-dom";
+import { CartCard, CheckoutCard } from "../../components";
+import { useCartContext } from "../../context/cart-context";
+import { useEffect } from "react";
+
+const Cart = () => {
+  const { cartState } = useCartContext();
+
+  useEffect(() => {
+    document.title = `(${cartState.cart.length}) Cart | Saiyan Merch`;
```
Comment (Ankur9669) [inline src/pages/Cart/Cart.jsx:11]: Suggestion: You can create a custom hook, but for the time being this is also fine

File: src/pages/Cart/cart.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+.empty-cart-title {
+  color: #ff527b;
+}
+
+.empty-cart-btn {
+  border: none;
+  padding: 0.6rem 1rem;
+  background-color: #ff527b;
```
Comment (Ankur9669) [inline src/pages/Cart/cart.css:69]: fix: Use css variable here

File: src/pages/Login/Login.jsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    document.title = "Login | Saiyan Merch";
+  }, []);
+
+  return (
+    <div className="login-form-container">
+      <div className="saiyan-form">
+        <h1 className="form__heading">Login</h1>
+
+        <div className="input-group">
+          <label className="login-form__label" for="email">
```
Comment (Ankur9669) [inline src/pages/Login/Login.jsx:16]: fix: use htmlFor here instead of for

File: src/pages/Login/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            Email adresses
+          </label>
+          <input
+            className="input"
+            type="email"
+            name="email"
+            id=""
```
Comment (Ankur9669) [inline src/pages/Login/Login.jsx]: also use id="email" here to connect label with input

File: src/pages/Login/Login.jsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+              <label for="checkbox">Remember me</label>
+            </span>
+
+            <span>
+              <a href="#">Forgot Password?</a>
+            </span>
+          </div>
+
+          <div className="actions">
+            <a className="btn btn--primary" href="#">
```
Comment (Ankur9669) [inline src/pages/Login/Login.jsx:46]: Can remove a tag from here

File: src/pages/Signup/Signup.jsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+            <Link className="btn btn--secondary" to="/login">
+              Already have an account
+            </Link>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
```
Comment (Ankur9669) [inline src/pages/Signup/Signup.jsx:77]: Same use htmlFor and id in input and labels

File: src/pages/Wishlist/Wishlist.jsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        </div>
+      ) : (
+        <>
+          <div className="wishlist__header__container">
+            <h2 className="cart__heading">My Wishlist</h2>
+          </div>
+
+          <div className="wishlist__card__container">
+            {wishlistState.wishlist.map((x) => (
+              <ProductCard products={x} />
```
Comment (Ankur9669) [inline src/pages/Wishlist/Wishlist.jsx]: fix: Use key here

**Review by Ankur9669 — APPROVED** [review]
Most of the coding conventions are looking fine, you can improve your code, i have given some suggestions after that you can merge

**Pull request merged**